game/src/pregame.rs

```diff
@@ -93,62 +93,87 @@ pub struct MainMenu {
 impl MainMenu {
     pub fn new(ctx: &mut EventCtx, app: &App) -> Box<dyn State<App>> {
         let col = vec![
-            Btn::svg_def("system/assets/pregame/quit.svg")
-                .build(ctx, "quit", Key::Escape)
-                .align_left(),
             {
                 let mut txt = Text::from(Line("A/B STREET").display_title());
                 txt.add(Line("Created by Dustin Carlino, Yuwen Li, & Michael Kirk"));
                 txt.draw(ctx).centered_horiz()
             },
-            Widget::row(vec![
-                Btn::svg(
-                    "system/assets/pregame/tutorial.svg",
-                    RewriteColor::Change(Color::WHITE, app.cs.hovering),
-                )
-                .tooltip({
-                    let mut txt = Text::tooltip(ctx, Key::T, "Tutorial");
-                    txt.add(Line("Learn how to play the game").small());
-                    txt
-                })
-                .build(ctx, "Tutorial", Key::T),
-                Btn::svg(
-                    "system/assets/pregame/sandbox.svg",
-                    RewriteColor::Change(Color::WHITE, app.cs.hovering),
-                )
-                .tooltip({
-                    let mut txt = Text::tooltip(ctx, Key::S, "Sandbox");
-                    txt.add(Line("No goals, try out any idea here").small());
-                    txt
-                })
-                .build(ctx, "Sandbox mode", Key::S),
-                Btn::svg(
-                    "system/assets/pregame/challenges.svg",
-                    RewriteColor::Change(Color::WHITE, app.cs.hovering),
-                )
-                .tooltip({
-                    let mut txt = Text::tooltip(ctx, Key::C, "Challenges");
-                    txt.add(Line("Fix specific problems").small());
-                    txt
-                })
-                .build(ctx, "Challenges", Key::C),
-            ])
+            Widget::row({
+                let btn_builder = app
+                    .cs
+                    .btn_primary_dark()
+                    .image_dims(ScreenDims::new(200.0, 100.0))
+                    .font_size(40)
```

Comment:
There are three places I've set custom sizes - here in these front screen buttons is one. This is pretty close to the size of the text that was previously baked into the SVG. We don't have a preset with that value, and AFAICT don't use it anywhere except on this screen. One nice thing about the builder is it can give you some local DRY-ness - the same `btn_builder` is used to build the three image buttons.

The speed panel has the other two custom font sizes. Getting all the disparate widgets to be the same height in the speed controls was really finicky. If you're off, because of the way `row` works, the other buttons in the row have their layout affected, so the text becomes uncentered, or, when hovering, the highlighted background doesn't cover the whole widget. So there I took some liberties to fudge things by a couple pixels to get things to line up.

I think in general, like you alluded to, we almost always want to use font-size from the presets to have a consistent hierarchy.